documentation/modules/exploit/winrm/winrm_script_exec.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+
+Windows Remote Management (WinRM) is a feature of Windows Vista that allows administrators to remotely run management scripts. It handles remote connections by means of the WS-Management Protocol, which is based on SOAP (Simple Object Access Protocol).   This module uses valid credentials to login to the WinRM service and execute a payload. It has two available methods for payload delivery: Powershell 2.0 and VBS CmdStager. This module will check if Poweshell 2.0 is available, and if so then it will use that method. Otherwise it falls back to the VBS CmdStager which is less stealthy.
```

Comment:
you could try it, but when you don't set a payload msf attempts to set a default compatible payload, so it more than likely just defaulted to something.